src/main/webapp/app/shared/modal/AddMutationModal.tsx

```diff
@@ -359,8 +361,9 @@ function AddMutationModal({
   }
 
   const fetchNormalAlterationDebounced = useCallback(
-    _.debounce((newAlteration: string, alterationIndex: number, alterationData: AlterationData[]) => {
-      fetchNormalAlteration(newAlteration, alterationIndex, alterationData);
+    _.debounce(async (newAlteration: string, alterationIndex: number, alterationData: AlterationData[]) => {
+      await fetchNormalAlteration(newAlteration, alterationIndex, alterationData);
```

Comment:
We should probably handle scenario when the async function fails. I can create a ticket to make sure that all async function errors are caught.

Reply:
Ok sounds good, we can come back through for more error checking.